site/en/blog/extension-news-october-2023/index.md

```diff
@@ -0,0 +1,151 @@
+- **Side Panel API** In [Chrome 116](https://chromiumdash.appspot.com/commit/8e7430446eaa0b80964b0ab1fd816ac6f33fd4cd) you can use [`sidepanel.open()`](/docs/extensions/reference/sidepanel/#user-interaction), to open the extension side panel programmatically in response to a user gesture, such as a context menu click.
+- **TabCapture API** Added the ability to call `getMediaStreamId()` from the extension service worker and obtain a [`MediaStream`](https://developer.mozilla.org/docs/Web/API/MediaStream) object from a stream ID in an offscreen document in Chrome 116. See [Audio recording and screen capture](/docs/extensions/mv3/screen_capture/) for examples.
+- **DeclarativeNetRequest API:** The default value for the [`isUrlFilterCaseSensitive`](/docs/extensions/reference/declarativeNetRequest/#property-RuleCondition-isUrlFilterCaseSensitive) property was changed to `false` in [Chrome 118](https://chromiumdash.appspot.com/commit/d90e6a56d0e77ce5d278a5b070098c5d8f7081fd).
+
+### Coming soon... {: #coming-soon-apis }
+
+Upcoming Chrome versions will release features that will complete the remaining items on the [known issues page](/docs/extensions/migrating/known-issues/#closing-the-platform-gap). Additionally, we plan to add the following features:
+
+- The **UserScripts API** will allow user script managers to coordinate how and when to inject a collection of user scripts into web pages. See the [WECG proposal](https://github.com/w3c/webextensions/blob/main/proposals/user-scripts-api.md) for details.
+- The **ReadingList API** will allow developers to create, read, update, and delete metadata located in the Reading List panel of the side panel. Watch [What's new in Chrome extensions](/docs/extensions/whatsnew/) for the announcement.
+- Following [feedback](https://github.com/w3c/webextensions/issues/318) in the Web Extensions Community Group, we are significantly increasing the limit on enabled static rulesets from 10 to 50. Additionally, we are increasing the total number of allowed static rulesets from 50 to 100. This is currently available in Canary.
+- The **File Handling API:** is will be available for ChromeOS extensions starting in ChromeOS 120, which lets extensions open files with specified MIME types and file extensions in a similar manner to web platform file handling.
+- Extensions will be able to use the web [Push API](https://developer.mozilla.org/docs/Web/API/Push_API) via [self.registration.pushManager.subscribe()](https://developer.mozilla.org/docs/Web/API/PushManager/subscribe) without showing a user-visible notification by setting `userVisibleOnly` to `false`. This will make push notifications a more seamless alternative to WebSockets in service workers (MV3) for asynchronous client-server communication. See [Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1319986) and [WECG discussion](https://github.com/w3c/webextensions/issues/208) for details.
```

Comment:
```suggestion
- Extensions will be able to use the web [Push API](https://developer.mozilla.org/docs/Web/API/Push_API) via [`self.registration.pushManager.subscribe()`](https://developer.mozilla.org/docs/Web/API/PushManager/subscribe) without showing a user-visible notification by setting `userVisibleOnly` to `false`. This will make push notifications a more seamless alternative to WebSockets in service workers (MV3) for asynchronous client-server communication. See [Chromium bug](https://bugs.chromium.org/p/chromium/issues/detail?id=1319986) and [WECG discussion](https://github.com/w3c/webextensions/issues/208) for details.
```